packages/amplify-category-auth/provider-utils/awscloudformation/index.js

```diff
@@ -154,4 +156,77 @@ function updateResource(context, category, service) {
     .then(() => props.resourceName);
 }
 
-module.exports = { addResource, updateResource };
+async function console(context, amplifyMeta) {
+  const cognitoOutput = getCognitoOutput(amplifyMeta);
+  if (cognitoOutput) {
+    const { Region } = amplifyMeta.providers.awscloudformation;
+    if (cognitoOutput.UserPoolId && cognitoOutput.IdentityPoolId) {
+      const answer = await inquirer.prompt({
+        name: 'selection',
+        type: 'list',
+        message: 'Which console',
+        choices: ['User Pool', 'Identity Pool', 'Both'],
+        default: 'Both',
+      });
+
+      switch (answer.selection) {
+        case 'User Pool':
+          await openUserPoolConsole(context, Region, cognitoOutput.UserPoolId);
+          break;
+        case 'Identity Pool':
+          await openIdentityPoolConsole(context, Region, cognitoOutput.IdentityPoolId);
+          break;
+        default:
+          await openUserPoolConsole(context, Region, cognitoOutput.UserPoolId);
+          await openIdentityPoolConsole(context, Region, cognitoOutput.IdentityPoolId);
+          break;
+      }
+    } else if (cognitoOutput.UserPoolId) {
+      await openUserPoolConsole(context, Region, cognitoOutput.UserPoolId);
+    } else {
+      await openIdentityPoolConsole(context, Region, cognitoOutput.IdentityPoolId);
+    }
+    context.print.info('');
+  } else {
+    context.print.error('Cognito resources have NOT been created for your project.');
```

Comment:
"Amazon Cognito service has not been added to your project"?

Reply:
If `amplify auth console` is executed after `amplify auth add` but before `amplify push`, then technically, Cognito has already been "added", but not "created". That's why I used "created". 

I can add  "Amazon" to the error message though.